backend/app/search/config.py

```diff
@@ -23,5 +23,16 @@ class IndexSettings:
         }
     }
 
+    metadata_mappings = {}
+    # "properties": {
```

Comment:
I removed the mappings because I noticed when the mapping are there for static fields, it somehow creating duplicate fields under a field "doc". So it looked like below:
{
    "metadata": {
        "aliases": {},
        "mappings": {
            "properties": {
                "contents": {
                    "type": "object"
                },
                "context": {
                    "type": "text"
                },
                "context_url": {
                    "type": "text"
                },
                "created": {
                    "type": "date"
                },
                "creator": {
                    "type": "keyword"
                },
                "doc": {
                    "properties": {
                        "contents": {
                            "properties": {
                                "alternateName": {
                                    "type": "text",
                                    "fields": {
                                        "keyword": {
                                            "type": "keyword",
                                            "ignore_above": 256
                                        }
                                    }
                                },
                                "latitude": {
                                    "type": "float"
                                },
                                "longitude": {
                                    "type": "float"
                                }
                            }
                        },
                        "context_url": {
                            "type": "text",
                            "fields": {
                                "keyword": {
                                    "type": "keyword",
                                    "ignore_above": 256
                                }
                            }
                        },
                        "created": {
                            "type": "date"
                        },
                        "creator": {
                            "type": "text",
                            "fields": {
                                "keyword": {
                                    "type": "keyword",
                                    "ignore_above": 256
                                }
                            }
                        },
                        "reource_type": {
                            "type": "text",
                            "fields": {
                                "keyword": {
                                    "type": "keyword",
                                    "ignore_above": 256
                                }
                            }
                        },
                        "resource_id": {
                            "type": "text",
                            "fields": {
                                "keyword": {
                                    "type": "keyword",
                                    "ignore_above": 256
                                }
                            }
                        }
                    }
                },
                "resource_id": {
                    "type": "text"
                },
                "resource_type": {
                    "type": "text"
                }
            }
        },
        "settings": {
            "index": {
                "routing": {
                    "allocation": {
                        "include": {
                            "_tier_preference": "data_content"
                        }
                    }
                },
                "number_of_shards": "5",
                "provided_name": "metadata",
                "creation_date": "1667489155177",
                "number_of_replicas": "5",
                "uuid": "g5MQr--BQB6q2B8boKZWkA",
                "version": {
                    "created": "8030399"
                }
            }
        }
    }
}

It made the search complicated. I removed the mappings and the index now looks like this:

{
    "metadata": {
        "aliases": {},
        "mappings": {
            "properties": {
                "doc": {
                    "properties": {
                        "contents": {
                            "properties": {
                                "alternateName": {
                                    "type": "text",
                                    "fields": {
                                        "keyword": {
                                            "type": "keyword",
                                            "ignore_above": 256
                                        }
                                    }
                                },
                                "latitude": {
                                    "type": "float"
                                },
                                "longitude": {
                                    "type": "float"
                                }
                            }
                        },
                        "context_url": {
                            "type": "text",
                            "fields": {
                                "keyword": {
                                    "type": "keyword",
                                    "ignore_above": 256
                                }
                            }
                        },
                        "created": {
                            "type": "date"
                        },
                        "creator": {
                            "type": "text",
                            "fields": {
                                "keyword": {
                                    "type": "keyword",
                                    "ignore_above": 256
                                }
                            }
                        },
                        "reource_type": {
                            "type": "text",
                            "fields": {
                                "keyword": {
                                    "type": "keyword",
                                    "ignore_above": 256
                                }
                            }
                        },
                        "resource_id": {
                            "type": "text",
                            "fields": {
                                "keyword": {
                                    "type": "keyword",
                                    "ignore_above": 256
                                }
                            }
                        }
                    }
                }
            }
        },
        "settings": {
            "index": {
                "routing": {
                    "allocation": {
                        "include": {
                            "_tier_preference": "data_content"
                        }
                    }
                },
                "number_of_shards": "5",
                "provided_name": "metadata",
                "creation_date": "1667507228722",
                "number_of_replicas": "5",
                "uuid": "WO4zd7trTGyf4P_2wGpPOw",
                "version": {
                    "created": "8030399"
                }
            }
        }
    }
}

I can now refer to the fields as doc.creator or doc.content.latitude etc. Sorry for long post, I hope it makes sense.